srcs/app/test/integration/server_name_test.py

```diff
@@ -0,0 +1,87 @@
+from pathlib import Path
+import pytest
+import requests
+import subprocess
+import time
+
+# Before running these tests add the following
+# to /etc/hosts:
+# - 127.0.0.1 example.com
+# - 127.0.0.1 example1.com
+# - 127.0.0.1 example2.com
```

Comment:
I'm not sure why, after this, to me locally delete integration test fails like:

```
alvaro@Tuor:~/webserv(feat/server_name)$ pytest --tb=short
======================================================================= test session starts =======================================================================
platform linux -- Python 3.8.10, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/alvaro/webserv
plugins: xonsh-0.9.13
collected 34 items                                                                                                                                                

srcs/app/test/integration/delete_method_test.py ....F..                                                                                                     [ 20%]
srcs/app/test/integration/get_method_test.py ...........                                                                                                    [ 52%]
srcs/app/test/integration/misc_test.py .                                                                                                                    [ 55%]
srcs/app/test/integration/post_method_test.py .......                                                                                                       [ 76%]
srcs/app/test/integration/server_name_test.py ........                                                                                                      [100%]

============================================================================ FAILURES =============================================================================
_______________________________________________________________ test_delete_forbidden_resource_403 ________________________________________________________________
srcs/app/test/integration/delete_method_test.py:92: in test_delete_forbidden_resource_403
    assert response.status_code == 403
E   assert 200 == 403
E    +  where 200 = <Response [200]>.status_code
===================================================================== short test summary info =====================================================================
FAILED srcs/app/test/integration/delete_method_test.py::test_delete_forbidden_resource_403 - assert 200 == 403
================================================================== 1 failed, 33 passed in 21.10s ==================================================================
alvaro@Tuor:~/webserv(feat/server_name)$ 
```

Reply:
I think it should work despite adding those domains to /etc/hosts thinking Maybe because you've fetched this new branch the folder no_privileges_folder has changed its ownership?

In spite of all this, as @gbudau says, I will remove those comments since Host header is setup for each single test.

You will still need to add those domains if want to play around with your browser. This is going to be tricky to test during correction since we don't have ownership of /etc/hosts in campus and the most visual way to test this is on a browser. We will have to use curl or something similar to demonstrate it.